packages/npm-dependencies-validation/src/utils/npmUtil.ts

```diff
@@ -0,0 +1,26 @@
+import { spawn } from "child_process";
+
+export function getNPM(): string {
+  return /^win/.test(process.platform) ? "npm.cmd" : "npm";
+}
+
+export function spawnCommand(
+  commandArgs: string[],
+  workingDir: string
```

Comment:
no need for two terms to describe the same things `cwd` vs `workingDir`

Reply:
Done